notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for station_dir in os.listdir(path_dict["InputPath"]):
</pre><p>If the <code>InputPath</code> includes previous runs, this will list all of those as well. lets reafctor this loop to use <code>station_names_list instead</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>